dev-packages/application-manager/package.json

```diff
@@ -41,7 +41,7 @@
     "font-awesome-webpack": "0.0.5-beta.2",
     "fs-extra": "^4.0.2",
     "ignore-loader": "^0.1.2",
-    "less": "^2.7.2",
+    "less": "^3.0.3",
```

Comment:
@marcdumais-work CQ?

Reply:
@akosyakov done.

Reply:
Not now. We decided to not hold back such changes until the foundation has decided how to go forward.